Assignment-11-September-4/vite-project/src/Components/Login/ForgetPassword.tsx

```diff
@@ -0,0 +1,69 @@
+import { useForm, SubmitHandler } from "react-hook-form";
+import { FormControl, Button, Box, TextField, Typography } from "@mui/material";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { useNavigate } from "react-router-dom";
+import { useAppDispatch, useAppSelector } from "../../store/store";
+import { sendVerificationEmail } from "../../store/loginThroughCodeSlice";
+import { useEffect } from "react";
+import { sendVerificationEmailRequest } from "../../sagaStore/sagas/codeSagaSlice";
+
+type FormValues = {
+  email: string;
+};
+
+const ForgetPassword: React.FC = () => {
```

Comment:
we always need to be consistant while coding, if we are using named functions with other components then why we use anonymous function here.